apps/finance/contracts/Finance.sol

```diff
@@ -22,6 +22,7 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
     uint64 constant public MAX_PAYMENTS_PER_TX = 20;
     uint64 constant public MAX_UINT64 = uint64(-1);
     uint256 constant public MAX_UINT = uint256(-1);
+    uint256 constant NO_TRANSACTION = 0;
```

Comment:
Let's make this explicitly internal, and maybe we could also add a constant for `NO_PAYMENT`?

Reply:
Fixed in d39b2da.